__init__.py

```diff
@@ -39,21 +39,28 @@
                 'external', 'netCDF4-win64'))
             import netCDF4
 
-            msg = 'Used netCDF4 library, provided with plugin. Python-netcdf version %{python-netcdf}s, '\
-                'netCDF4 version %{netcdf)s and HDF5 version %{netcdf)s.'% {
-                'python-netcdf': netCDF4.__version__,
-                'netcdf': netCDF4.__netcdf4libversion__,
-                'hdf5': netCDF4.__hdf5libversion__
-            }
-            log(msg)
-            print msg
+            msg += 'Used netCDF4 library, provided with plugin. '
         else:
-            pop_up_info('Error: could not find netCDF4 installation. Change to the 64-bit vresion of QGIS or try to '
-                        'install the netCDF4 python libary yourself.')
+            pop_up_info('Error: could not find netCDF4 installation. Change '
+                        'to the 64-bit vresion of QGIS or try to install the '
+                        'netCDF4 python libary yourself.')
+            netCDF4 = None
     else:
-        pop_up_info('Error: could not find netCDF4 installation. Please install python-netCDF4 package.')
+        pop_up_info('Error: could not find netCDF4 installation. Please '
+                    'install python-netCDF4 package.')
+        netCDF4 = None
+
+if netCDF4 is not None:
+    msg += 'Python-netcdf version {python_netcdf}, netCDF4 version ' \
+           '{netcdf} and HDF5 version {netcdf}.'.format(
+                    python_netcdf = netCDF4.__version__,
+                    netcdf = netCDF4.__netcdf4libversion__,
+                    hdf5 = netCDF4.__hdf5libversion__)
+
+    log(msg)
+    print msg
```

Comment:
a) bij de GGMN plugin had ik er last van dat ik alle "print" statements eruit moest gooien omdat sommige installaties in Thailand er last van hadden. Jan-Maarten even vragen. Even als "heads-up" dus dat dit misschien een probleem kan worden, raar genoeg.

b) Beter overal gelijk `print(msg)` gebruiken in plaats van `print msg`. Dan is het python3-compatibeler.